app/serializers/rest/report_note_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class REST::ReportNoteSerializer < ActiveModel::Serializer
+  attributes :id, :content, :created_at
+
+  has_one :report, serializer: REST::ReportSerializer
```

Comment:
Really we don't need the full report here, just the Report ID and the URL for the Report Note (I don't think we should be constructing the URL on the client like I have).